ibm/service/power/resource_ibm_pi_cloud_connection.go

```diff
@@ -200,8 +205,9 @@ func resourceIBMPICloudConnectionCreate(ctx context.Context, d *schema.ResourceD
 			vpcIds := flex.ExpandStringList(v.(*schema.Set).List())
 			vpcs := make([]*models.CloudConnectionVPC, len(vpcIds))
 			for i, vpcId := range vpcIds {
+				vpcIdCopy := vpcId[0:]
```

Comment:
```suggestion
				vpcId := vpcId
```
This single line change should work here.

Reply:
I think setting the variable to itself is confusing.

Reply:
I see it is recommended easier approach for such cases. Ref: http://golang.org/doc/faq#closures_and_goroutines

Reply:
I don't read that as recommended but rather easier than their other example of approaching this scenario which was creating an anonymous function that would use a copy of the string so it will not be overridden. So can I keep this version since even that webpage says it seems odd? Thanks.

Reply:
I don't see any issue with this approach.